plasmapy/diagnostics/thomson.py

```diff
@@ -68,62 +69,62 @@ def spectral_density_lite(
     The :term:`lite-function` version of
     `~plasmapy.diagnostics.thomson.spectral_density`.  Performs the same
     thermal speed calculations as
-    `~plasmapy.diagnostics.thomson.spectral_density`, but is intended for
-    computational use and, thus, has data conditioning safeguards
+    `~plasmapy.diagnostics.thomson.spectral_density`, but is intended
+    for computational use and thus has data conditioning safeguards
     removed.
 
     Parameters
     ----------
-    wavelengths : `~numpy.ndarray`, shape (Nwavelengths,)
-        Array of wavelengths in meters over which the spectral density function
-        will be calculated.
+    wavelengths : (Nλ,) `~numpy.ndarray`
```

Comment:
Using a docstring convention from matplotlib's dev guide for the size/shape of parameters, which is also included in #1757.